testsuite/features/step_definitions/common_steps.rb

```diff
@@ -142,12 +142,16 @@
   # same name in the events history - however, that's the best we have so far.
   steps %(
     When I follow "Events"
+    And I wait until I see "Pending Events" text
```

Comment:
This first step might raise a timeout depending the state and the tab you are isn't it?

Reply:
From my tests, clicking on Events tab always bring you to Pending sub-tab. I will keep an eye on this